src/utils/clean.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+import numpy as np
+import pandas as pd
+
+
+def no_nonsense(df):
+    """
+    TODO: this function should resolve issue #9
+    NOTE: It is possible that this function is not necessary.
+        if that is the case, your PR should be to remove this function
+        and the call to this function from this script.
+    https://github.com/the4thparadigm/denvercrime/issues/9
+    TODO: write a docstring explaining your approach
+    NOTE: Do not hesitate to add other functions if you find it appropriate.
+        Be sure to add calls to any additional functions either here or in 
+        main().
+    TODO: submit a PR with your changes to this file
+
+    Preconditions:
+        no_incomplete() has already been called on df.
+
+    Parameters:
+        df (DataFrame): crime data
+
+    Returns:
+        (DataFrame): crime data without nonsensical events
+    """
+    return df
+
+def no_incomplete(df):
+    """
+    TODO: this function should resolve issue #8
+    https://github.com/the4thparadigm/denvercrime/issues/8
+    TODO: write a docstring explaining your approach
+    NOTE: Do not hesitate to add other functions if you find it appropriate.
+        Be sure to add calls to any additional functions either here or in 
+        main().
+    TODO: submit a PR with your changes to this file
+
+    Parameters:
+        df (DataFrame): crime data
+
+    Returns:
+        (DataFrame): crime data without incomplete events
+    """
+    return df
+
+
+def main():
+    """ Entry point.
+    Calls no_nonsense() and no_incomplete() and saves the
+    result in data/processed/cleaned.csv
+    """
+    crime = pd.read_csv('data/raw/crime.csv')
```

Comment:
This assumes project root is in the `PYTHONPATH` env variable, right? I'm okay with that assumption, but we may have to work with folks to set the env variable. Alternately, when developing people can have a temporary full path version that loads the data from their system, then remove it before the PR

Reply:
Either they have the project in the PYTHONPATH or they are running the script from the project root. But it shouldn't be too hard to set the PYTHONPATH with them, so we can do that real quick at the beginning of the meeting.